---
output:
  bookdown::pdf_book:
    toc : no
    keep_tex: no
    number_sections: yes
---

<!-- Remember, It can't be longer than 350 words! -->

Fibrotic interstitial lung disease (ILD) comprises a heterogeneous group of >200 disorders that are common in older adults and characterized by excessive deposition of fibrous tissue in the lung interstitium, resulting in decreased blood uptake of oxygen. A critical aspect of ILD research that must be addressed is the difficulty in correctly diagnosing specific ILD subtypes, especially when differentiating between common subtypes such as idiopathic pulmonary fibrosis (IPF) and hypersensitivity pneumonitis (HP). IPF requires different pharmacotherapy compared to non-IPF ILDs, but IPF is difficult to conclusively diagnose due to non-specific radiological patterns and risks associated with histopathological lung biopsies. Additionally, certain ILD subtypes present with rapid decline in lung function known as progressive pulmonary fibrosis (PPF), and little is known about the biology behind this phenotype. Given the risk of mortality associated with lung biopsies in older adults, peripheral biomarkers taken from blood samples are an attractive medium of investigation. 

The objective of this thesis was to investigate the peripheral transcriptome for biomarkers of ILD diagnosis and progression. To establish a baseline of changes in gene expression in ILD, we performed a systematic review and meta-analysis of the bulk ILD transcriptome in both lung and blood samples, and identified unique and shared pathways between ILD subtypes. Through an unsupervised analysis, molecular endotypes of various disease mechanisms were identified and associated with lung function. In a pilot study, the feasbility of using whole blood RNA to differentiate between ILD subtypes and relate their expression to the lung microenvironment was examined in a small cohort using a targeted gene expression assay, and we identified blood samples from systemic sclerosis-associated ILD (SSc-ILD) as differentially expressed from IPF and HP. This was confirmed in a study of ILD subtypes using whole genome sequencing, where a significant increase in interferon-associated genes and genes associated with progression were observed in SSc-ILD. Using ILD gene expression data, we further identified a model predicting survival which was validated on external IPF samples. Finally, cell-specific transcriptomic profiles were identified in IPF and HP using single cell RNA-sequencing, which were associated with previously identified fibrotic cell types in the lung. 