---
output:
  bookdown::pdf_book:
    toc : no
    keep_tex: no
    number_sections: yes
---

```{r echo=FALSE}
library(readxl) 
library(dplyr)
read_excel_allsheets <- function(filename, tibble = FALSE) {
    sheets <- readxl::excel_sheets(filename)
    x <- lapply(sheets, function(X) readxl::read_excel(filename, sheet = X))
    if(!tibble) x <- lapply(x, as.data.frame)
    names(x) <- sheets
    x
}
# function for sig figs
round_to_sigfigs <- function(x, sigfigs) {
  if (!is.numeric(x)) return(x)  
  round(x, digits = sigfigs)
}
ensembl <- readRDS(paste0(getwd(), "/Tables/ensembl_240412.RDS"))
```

<!-- This section is mandatory! -->
\captionsetup{width=6.5in} <!-- makes table caption stays in margins -->

### Differential expression

(ref:rnaseqPCA-cap) \textbf{Principal component analysis (PCA) of blood RNA-seq cohort.} (A) PCA plot of blood RNA-seq samples. (B) Correlation of PCA components (PCs) with demographic and clinical metadata. Values shown are Pearson correlation coefficients. 

```{r rnaseqPCA, eval = T, out.width='0.7\\linewidth',echo=FALSE, fig.cap="(ref:rnaseqPCA-cap)", fig.scap = "Correlation of PCs with metadata (blood RNA-seq)", out.extra='', fig.align='center'}
knitr::include_graphics("./Figures/BloodRNAseq/bloodRNAseq_pca_v2.pdf")
```

\begin{singlespace}
\pagebreak

(ref:ipfvsssc-cap) \textbf{List of top 50 differentially expressed genes between IPF and SSc-ILD.} Results shown are adjusted for race. 

```{r ipfvsssc, eval = T, echo = F, warning = FALSE, message=FALSE, out.extra='', fig.scap='DEGs: IPF vs SSc-ILD', fig.align='center'}
deg_table <- read_excel_allsheets(here::here('Tables', 'BloodRNAseq', "DEG_diagnosis_race_240529.xlsx"))
IPFSSc <- deg_table[["IPFvsSSc"]]
gene_key <- biomaRt::getBM(attributes=c('ensembl_gene_id', 'chromosome_name'), 
      filters = 'ensembl_gene_id', 
      values = IPFSSc$`ensembl_gene_id`,
      mart = ensembl)

IPFSSc <- IPFSSc %>%
  filter(!ensembl_gene_id %in% (gene_key %>% filter(chromosome_name =="Y") %>% pull(ensembl_gene_id))) %>%
  mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
  mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
  dplyr::select(ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val)

colnames(IPFSSc) <- c("Ensembl ID","Gene","(ref:logfc)", "(ref:p-it)-value", "FDR")

kableExtra::kbl(
  IPFSSc %>% head(n=50),
  longtable = T,
  format = 'latex',
  booktabs = T, 
  #linesep = "",
  align = "llrcc",
  caption="(ref:ipfvsssc-cap)") %>%
  column_spec(c(1), width="1.2in") %>%
  column_spec(c(2), width="0.7in") %>%
  column_spec(c(3), width="0.6in") %>%
  column_spec(c(4), width="0.6in") %>%
  column_spec(c(5), width="0.6in") %>%
  kableExtra::kable_styling(full_width = F) %>%
  #kableExtra::kable_styling(latex_options = "scale_down") #%>% 
  kableExtra::kable_styling(latex_options = c("HOLD_position", "repeat_header"), font_size = 8) # ensures the table isn't "floated" and pins it to the current location
```

\pagebreak

(ref:ipfsscpathway-cap) \textbf{Top 20 pathways associated with up- and down-regulated SSc-ILD vs IPF differentially expressed genes.} 

```{r ipfsscpathway, eval = T, echo = F, warning = FALSE, message=FALSE, out.extra='', fig.scap='DEGs: IPF vs SSc-ILD', fig.align='center'}
sscvsipf_up <- read_xlsx(here::here('Tables', 'BloodRNAseq', "SScvsIPF_up_pathways.xlsx"))
sscvsipf_down <- read_xlsx(here::here('Tables', 'BloodRNAseq', "SScvsIPF_down_pathways.xlsx"))
sscvsipf_up <- sscvsipf_up %>% filter(subcollection %in% c( "GO Biological Process", "Hallmark", "Reactome Pathways", "KEGG Pathways") | collection %in% c("TISSUES"))
sscvsipf_down <- sscvsipf_down %>% filter(subcollection %in% c( "GO Biological Process", "Hallmark", "Reactome Pathways", "KEGG Pathways") | collection %in% c("TISSUES"))

kableExtra::kbl(
  rbind(
    sscvsipf_up %>% mutate(Direction="SSc-ILD Up") %>% dplyr::select(Direction, geneset, fdr) %>% slice(1:20),
    sscvsipf_down %>% mutate(Direction="SSc-ILD Down") %>% dplyr::select(Direction, geneset, fdr) %>% slice(1:20)
      ) %>% 
    mutate(fdr=formatC(fdr, format = "e", digits=2)) %>%
    mutate(geneset = gsub("_", " ", geneset)) %>%
    rename(Pathway = geneset) %>% rename(FDR=fdr),
  longtable = T,
  format = 'latex',
  booktabs = T, 
  #linesep = "",
  align = "llc",
  caption="(ref:ipfsscpathway-cap)") %>%
  column_spec(column=1, width="1.0in") %>%
  column_spec(column=2, width="4.5in") %>%
  kableExtra::kable_styling(full_width = F) %>%
  #kableExtra::kable_styling(latex_options = "scale_down") #%>% 
  kableExtra::kable_styling(latex_options = c("HOLD_position", "repeat_header"), font_size = 8) # ensures the table isn't "floated" and pins it to the current location
```

\pagebreak

(ref:ILDdeg-cap) \textbf{List of top 5 differentially expressed genes between various ILD subtypes.} Results shown are adjusted for race. 

```{r ILDdeg, eval = T, echo = F, warning = FALSE, message=FALSE, out.extra='', fig.scap='DEGs: IPF vs SSc-ILD', fig.align='center'}
ild_deg <- rbind(
  deg_table[["IPFvsHP"]] %>% 
    mutate(Comparison = "IPF vs HP") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5),
  deg_table[["HPvsSScILD"]] %>% 
    mutate(Comparison = "HP vs SSc-ILD") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5),  
  deg_table[["IPFvsIPAF"]] %>% 
    mutate(Comparison = "IPF vs IPAF") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5),  
  deg_table[["SScILDvsIPAF"]] %>% 
    mutate(Comparison = "SSc-ILD vs IPAF") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5),  
  deg_table[["IPFvsRA"]] %>% 
    mutate(Comparison = "IPF vs RA-ILD") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5),  
  deg_table[["HPvsRA"]] %>% 
    mutate(Comparison = "HP vs RA-ILD") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5),  
  deg_table[["SScILDvsRA"]] %>% 
    mutate(Comparison = "SSc-ILD vs RA-ILD") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5)
)

colnames(ild_deg) <- c("Comparison","Ensembl ID","Gene","(ref:logfc)", "(ref:p-it)-value", "FDR")

kableExtra::kbl(
  ild_deg,
  longtable = T,
  format = 'latex',
  booktabs = T, 
  #linesep = "",
  align = "llrcc",
  caption="(ref:ILDdeg-cap)") %>%
  column_spec(c(1), width="1.3in") %>%
  column_spec(c(2), width="1.2in") %>%
  column_spec(c(3), width="0.9in") %>%
  column_spec(c(4), width="0.6in") %>%
  column_spec(c(5), width="0.6in") %>%
  column_spec(c(6), width="0.6in") %>%
  kableExtra::kable_styling(full_width = F) %>%
  #kableExtra::kable_styling(latex_options = "scale_down") #%>% 
  kableExtra::kable_styling(latex_options = c("HOLD_position", "repeat_header"), font_size = 8)  # ensures the table isn't "floated" and pins it to the current location
```

\pagebreak

(ref:degrad-cap) \textbf{List of top 10 differentially expressed genes between radiological patterns.} Results shown are adjusted for race. 

```{r degrad, eval = T, echo = F, warning = FALSE, message=FALSE, out.extra='', fig.scap='DEGs: IPF vs SSc-ILD', fig.align='center'}
rad_table <- read_excel_allsheets(here::here('Tables', 'BloodRNAseq', "DEG_radiology_race_240529.xlsx"))

rad_deg <- rbind(
   rad_table[["UIPvsHP"]] %>% 
    mutate(Comparison = "UIP vs fHP") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=10),
  rad_table[["UIPvsNSIP"]] %>% 
    mutate(Comparison = "UIP vs NSIP") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=10),  
  rad_table[["HPvsNSIP"]] %>% 
    mutate(Comparison = "fHP vs NSIP") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=10) 
)

colnames(rad_deg) <- c("Comparison","Ensembl ID","Gene","(ref:logfc)", "(ref:p-it)-value", "FDR")

kableExtra::kbl(
  rad_deg,
  longtable = T,
  format = 'latex',
  booktabs = T, 
  #linesep = "",
  align = "lllrcc",
  caption="(ref:degrad-cap)") %>%
  column_spec(c(1), width="1.3in") %>%
  column_spec(c(2), width="1.2in") %>%
  column_spec(c(3), width="0.7in") %>%
  column_spec(c(4), width="0.6in") %>%
  column_spec(c(5), width="0.6in") %>%
  column_spec(c(6), width="0.6in") %>%
  kableExtra::kable_styling(full_width = F) %>%
  #kableExtra::kable_styling(latex_options = "scale_down") #%>% 
  kableExtra::kable_styling(latex_options = c("HOLD_position", "repeat_header"), font_size = 8)  # ensures the table isn't "floated" and pins it to the current location
```
\end{singlespace}

\pagebreak

(ref:ppfdeg-cap) \textbf{Differentially expressed genes in PPF.} Volcano plots of (A) PPF, (B) relative decline in FVC% predicted of >10\%, (C) radiological progression, and (D) relative decline in FVC% predicted of >5\% within 2 years. (E-G) Boxplots of significant DEGs associated with various PPF criteria. 

```{r ppfdeg, eval = T, out.width='0.80\\linewidth',echo=FALSE, fig.cap="(ref:ppfdeg-cap)", fig.scap = "DEGs: PPF", out.extra='', fig.align='center'}
knitr::include_graphics("./Figures/BloodRNAseq/ppf_deg_v1.pdf")
```

(ref:ppfdegipf-cap) \textbf{Differentially expressed genes in IPF patients that experience PPF.} (A) Volcano plot of DEGs between PPF and non-PPF patients in IPF, and (B) boxplots of THBS3-AS1 expression in ILD patients grouped by whether or not they experienced a relative decline in FVC\% predicted of >10\% within 2 years. 

```{r ppfdegipf, eval = T, out.width='0.75\\linewidth',echo=FALSE, fig.cap="(ref:ppfdegipf-cap)", fig.scap = "DEGs: radiological progression", out.extra='', fig.align='center'}
knitr::include_graphics("./Figures/BloodRNAseq/ppf_ipf_deg_v1.pdf")
```

\begin{singlespace}
\pagebreak

(ref:ppfdegtab-cap) \textbf{List of top 5 differentially expressed genes between patients classified as having PPF or not using INBUILD criteria.} Results shown are adjusted for race. 

```{r ppfdegtab, eval = T, echo = F, warning = FALSE, message=FALSE, out.extra='', fig.scap='DEG Table: PPF', fig.align='center'}
ppf <- read_excel_allsheets(here::here('Tables', 'BloodRNAseq', "DEG_INBUILD_240529.xlsx"))

ppf_table <- rbind(
   ppf[["PFILD_all"]] %>% 
    mutate(Comparison = "All ILD") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5),
  ppf[["PFILD_IPF_race"]] %>% 
    mutate(Comparison = "IPF") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5),  
  ppf[["PFILD_SScILD_race"]] %>% 
    mutate(Comparison = "SSc-ILD") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5) ,
  ppf[["PFILD_NSIP_race"]] %>% 
    mutate(Comparison = "NSIP") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5) 
)

colnames(ppf_table) <- c("Stratification","Ensembl ID","Gene","(ref:logfc)", "(ref:p-it)-value", "FDR")

kableExtra::kbl(
  ppf_table,
  longtable = T,
  format = 'latex',
  booktabs = T, 
  #linesep = "",
  align = "lllrcc",
  caption="(ref:ppfdegtab-cap)") %>%
  column_spec(c(1), width="1.0in") %>%
  column_spec(c(2), width="1.2in") %>%
  column_spec(c(3), width="0.8in") %>%
  column_spec(c(4), width="0.6in") %>%
  column_spec(c(5), width="0.6in") %>%
  column_spec(c(6), width="0.6in") %>%
  kableExtra::kable_styling(full_width = F) %>%
  #kableExtra::kable_styling(latex_options = "scale_down") #%>% 
  kableExtra::kable_styling(latex_options = c("HOLD_position", "repeat_header"), font_size = 8)  # ensures the table isn't "floated" and pins it to the current location
```

(ref:ppfradtab-cap) \textbf{List of top 5 differentially expressed genes between patients that do and do not experience radiological progression within 2 years.} Results shown are adjusted for race. 

```{r ppfradtab, eval = T, echo = F, warning = FALSE, message=FALSE, out.extra='', fig.scap='DEG Table: Radiological Progression', fig.align='center'}
rad_all <- read_excel_allsheets(here::here('Tables', 'BloodRNAseq', "DEG_rad_progression_2yr_240529.xlsx"))
rad_ipf <- read_excel_allsheets(here::here('Tables', 'BloodRNAseq', "DEG_rad_progression_2yr_IPF_240529.xlsx"))
rad_ssc <- read_excel_allsheets(here::here('Tables', 'BloodRNAseq', "DEG_rad_progression_2yr_SScILD_240529.xlsx"))
rad_nsip <- read_excel_allsheets(here::here('Tables', 'BloodRNAseq', "DEG_rad_progression_2yr_NSIP_240529.xlsx"))
  
rad_prog_table <- rbind(
   rad_all[["rad_progress_race"]] %>% 
    mutate(Comparison = "All ILD") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5),
  rad_ipf[["rad_progression_race_IPF"]] %>% 
    mutate(Comparison = "IPF") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5),  
  rad_ssc[["rad_progression_race_SScILD"]] %>%
    mutate(Comparison = "SSc-ILD") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5) ,
  rad_nsip[["rad_progression_race_NSIP"]] %>% 
    mutate(Comparison = "NSIP") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5) 
)

colnames(rad_prog_table) <- c("Stratification","Ensembl ID","Gene","(ref:logfc)", "(ref:p-it)-value", "FDR")

kableExtra::kbl(
  rad_prog_table,
  longtable = T,
  format = 'latex',
  booktabs = T, 
  #linesep = "",
  align = "lllrcc",
  caption="(ref:ppfradtab-cap)") %>%
  column_spec(c(1), width="1.0in") %>%
  column_spec(c(2), width="1.2in") %>%
  column_spec(c(3), width="0.8in") %>%
  column_spec(c(4), width="0.6in") %>%
  column_spec(c(5), width="0.6in") %>%
  column_spec(c(6), width="0.6in") %>%
  kableExtra::kable_styling(full_width = F) %>%
  #kableExtra::kable_styling(latex_options = "scale_down") #%>% 
  kableExtra::kable_styling(latex_options = c("HOLD_position", "repeat_header"), font_size = 8)  # ensures the table isn't "floated" and pins it to the current location
```

\end{singlespace}
\pagebreak

### Survival

\begin{singlespace}

(ref:ildsurvivalgenes-cap) \textbf{ILD survival model genes.} 

```{r ildsurvivalgenes, eval = T, echo = F, warning = FALSE, message=FALSE, out.extra='', fig.scap='ILD survival model genes', fig.align='center'}
ild_genes <- read.csv(here::here('Tables', 'BloodRNAseq', "ild_survival_genes.csv"), row.names=1)
ild_genes <- ild_genes %>% mutate_at(c("Overall"), round_to_sigfigs, sigfigs = 3) %>%
  dplyr::select(ensembl_gene_id, hgnc_symbol, Overall)
colnames(ild_genes) <- c("Ensembl ID", "Gene" , "Weight")

kableExtra::kbl(
  ild_genes,
  longtable = T,
  format = 'latex',
  booktabs = T, 
  linesep = "",
  align = "llc",
  caption="(ref:ildsurvivalgenes-cap)") %>%
  column_spec(c(1), width="1.2in") %>%
  column_spec(c(2), width="1.0in") %>%
  column_spec(c(3), width="0.6in") %>%
  kableExtra::kable_styling(full_width = F) %>%
  #kableExtra::kable_styling(latex_options = "scale_down") #%>% 
  kableExtra::kable_styling(latex_options = c("HOLD_position", "repeat_header"), font_size = 8)  # ensures the table isn't "floated" and pins it to the current location
```

(ref:notsscsurvivalgenes-cap) \textbf{ILD (excluding SSc-ILD) survival model genes.} 

```{r notsscsurvivalgenes, eval = T, echo = F, warning = FALSE, message=FALSE, out.extra='', fig.scap='ILD survival (excluding SSc-ILD) model genes', fig.align='center'}
ild_genes <- read.csv(here::here('Tables', 'BloodRNAseq', "ildnotssc_survival_genes.csv"), row.names=1)
ild_genes <- ild_genes %>% mutate_at(c("Overall"), round_to_sigfigs, sigfigs = 3) %>%
  dplyr::select(ensembl_gene_id, hgnc_symbol, Overall)
colnames(ild_genes) <- c("Ensembl ID", "Gene" , "Weight")

kableExtra::kbl(
  ild_genes,
  longtable = T,
  format = 'latex',
  booktabs = T, 
  linesep = "",
  align = "lrl",
  caption="(ref:notsscsurvivalgenes-cap)") %>%
  column_spec(c(1), width="1.0in") %>%
  column_spec(c(2), width="0.8in") %>%
  column_spec(c(3), width="0.6in") %>%
  kableExtra::kable_styling(full_width = F) %>%
  #kableExtra::kable_styling(latex_options = "scale_down") #%>% 
  kableExtra::kable_styling(latex_options = c("HOLD_position", "repeat_header"), font_size = 8) # ensures the table isn't "floated" and pins it to the current location
```

\end{singlespace}

(ref:notsscsurvival-cap) \textbf{ILD (excluding SSc-ILD) survival model derived from the peripheral transcriptome.} A survival model derived using 50x repeated 10-fold cross-validation was trained on our ILD cohort excluding SSc-ILD samples, with risk groups stratified by optimized cutpoints (A). The model was validated on two external IPF datasets, GSE28042 and GSE93606, which are shown with different risk group stratification based on splitting the hazard ratios into tertiles (B) or using optimized cutpoints (C). Log-rank \textit{P}-values shown in black are calculated between high and low risk groups, while the \textit{P}-value shown in grey is calculated between all three risk groups in (B). Selected genes in the model stratified by high and low expression are shown in (D). 

```{r notsscsurvival, eval = T, out.width='0.86\\linewidth',echo=FALSE, fig.cap="(ref:notsscsurvival-cap)", fig.scap = "ILD (excluding SSc-ILD) survival model", out.extra='', fig.align='center'}
knitr::include_graphics("./Figures/BloodRNAseq/PPF_survival_notSSc_v2.pdf")
```

(ref:herazomaya-cap) \textbf{Performance of the 52-gene risk profile published by Herazo-Maya \textit{et al.}} The 52-gene risk profile was calculated for the blood RNA-seq cohort (A) and validation cohorts (B), with the performance of proposed survival model on individual validation cohorts shown in (C). 

```{r herazomaya, eval = T, out.width='0.86\\linewidth',echo=FALSE, fig.cap="(ref:herazomaya-cap)", fig.scap = "Comparison with 52-gene risk model", out.extra='', fig.align='center'}
knitr::include_graphics("./Figures/BloodRNAseq/herazomaya_comp_v1.pdf")
```


