---
output:
  bookdown::pdf_book:
    toc : no
    keep_tex: no
    number_sections: yes
---

```{r echo=FALSE}
library(readxl) 
library(dplyr)
read_excel_allsheets <- function(filename, tibble = FALSE) {
    sheets <- readxl::excel_sheets(filename)
    x <- lapply(sheets, function(X) readxl::read_excel(filename, sheet = X))
    if(!tibble) x <- lapply(x, as.data.frame)
    names(x) <- sheets
    x
}
# function for sig figs
round_to_sigfigs <- function(x, sigfigs) {
  if (!is.numeric(x)) return(x)  
  round(x, digits = sigfigs)
}
ensembl <- readRDS(paste0(getwd(), "/Tables/ensembl_240412.RDS"))
```


<!-- This section is mandatory! -->
\captionsetup{width=6.5in} <!-- makes table caption stays in margins -->

## Differential expression

(ref:rnaseqPCA-cap) \textbf{Principal component analysis (PCA) of blood RNA-seq cohort.} (A) PCA plot of blood RNA-seq samples. (B) Correlation of PCA components (PCs) with demographic and clinical metadata. Values shown are Pearson correlation coefficients. 

```{r rnaseqPCA, eval = T, out.width='0.7\\linewidth',echo=FALSE, fig.cap="(ref:rnaseqPCA-cap)", fig.scap = "Correlation of PCs with metadata (blood RNA-seq)", out.extra='', fig.align='center'}
knitr::include_graphics("./Figures/BloodRNAseq/bloodRNAseq_pca_v2.pdf")
```

(ref:ipfvsssc-cap) \textbf{List of top 50 differentially expressed genes between IPF and SSc-ILD.} Results shown are adjusted for race. 

```{r ipfvsssc, eval = T, echo = F, warning = FALSE, message=FALSE, out.extra='', fig.scap='DEGs: IPF vs SSc-ILD', fig.align='center'}
deg_table <- read_excel_allsheets(here::here('Tables', 'BloodRNAseq', "DEG_diagnosis_race_240529.xlsx"))
IPFSSc <- deg_table[["IPFvsSSc"]]
gene_key <- biomaRt::getBM(attributes=c('ensembl_gene_id', 'chromosome_name'), 
      filters = 'ensembl_gene_id', 
      values = IPFSSc$`ensembl_gene_id`,
      mart = ensembl)

IPFSSc <- IPFSSc %>%
  filter(!ensembl_gene_id %in% (gene_key %>% filter(chromosome_name =="Y") %>% pull(ensembl_gene_id))) %>%
  mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
  mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
  dplyr::select(ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val)

colnames(IPFSSc) <- c("Ensembl ID","Gene","(ref:logfc)", "(ref:p-it)-value", "FDR")

kableExtra::kbl(
  IPFSSc %>% head(n=50),
  longtable = F,
  format = 'latex',
  booktabs = T, 
  linesep = "",
  align = "llrcc",
  caption="(ref:ipfvsssc-cap)") %>%
  kableExtra::kable_styling(full_width = TRUE) %>%  # table spans across left to right margins
  column_spec(c(1), width="1.2in") %>%
  column_spec(c(2), width="0.7in") %>%
  column_spec(c(3), width="0.6in") %>%
  column_spec(c(4), width="0.6in") %>%
  column_spec(c(5), width="0.6in") %>%
  #kableExtra::kable_styling(latex_options = "scale_down") #%>% 
  kableExtra::kable_styling(latex_options = c("hold_position"), font_size = 8) # ensures the table isn't "floated" and pins it to the current location
```

(ref:ipfsscpathway-cap) \textbf{Top 20 pathways associated with up- and down-regulated SSc-ILD vs IPF differentially expressed genes.} 

```{r ipfsscpathway, eval = T, echo = F, warning = FALSE, message=FALSE, out.extra='', fig.scap='DEGs: IPF vs SSc-ILD', fig.align='center'}
sscvsipf_up <- read_xlsx(here::here('Tables', 'BloodRNAseq', "SScvsIPF_up_pathways.xlsx"))
sscvsipf_down <- read_xlsx(here::here('Tables', 'BloodRNAseq', "SScvsIPF_down_pathways.xlsx"))
sscvsipf_up <- sscvsipf_up %>% filter(subcollection %in% c( "GO Biological Process", "Hallmark", "Reactome Pathways", "KEGG Pathways") | collection %in% c("TISSUES"))
sscvsipf_down <- sscvsipf_down %>% filter(subcollection %in% c( "GO Biological Process", "Hallmark", "Reactome Pathways", "KEGG Pathways") | collection %in% c("TISSUES"))

kableExtra::kbl(
  rbind(
    sscvsipf_up %>% mutate(Direction="SSc-ILD Up") %>% dplyr::select(Direction, geneset, fdr) %>% slice(1:20),
    sscvsipf_down %>% mutate(Direction="SSc-ILD Down") %>% dplyr::select(Direction, geneset, fdr) %>% slice(1:20)
      ) %>% 
    mutate(fdr=formatC(fdr, format = "e", digits=2)) %>%
    mutate(geneset = gsub("_", " ", geneset)) %>%
    rename(Pathway = geneset) %>% rename(FDR=fdr),
  longtable = F,
  format = 'latex',
  booktabs = T, 
  linesep = "",
  align = "llc",
  caption="(ref:ipfsscpathway-cap)") %>%
  kableExtra::kable_styling(full_width = TRUE) %>%  # table spans across left to right margins
  column_spec(column=1, width="1.0in") %>%
  column_spec(column=2, width="4.5in") %>%
  #kableExtra::kable_styling(latex_options = "scale_down") #%>% 
  kableExtra::kable_styling(latex_options = c("hold_position"), font_size = 8) # ensures the table isn't "floated" and pins it to the current location
```


(ref:ILDdeg-cap) \textbf{List of top 5 differentially expressed genes between various ILD subtypes.} Results shown are adjusted for race. 

```{r ILDdeg, eval = T, echo = F, warning = FALSE, message=FALSE, out.extra='', fig.scap='DEGs: IPF vs SSc-ILD', fig.align='center'}
ild_deg <- rbind(
  deg_table[["IPFvsHP"]] %>% 
    mutate(Comparison = "IPF vs HP") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5),
  deg_table[["HPvsSScILD"]] %>% 
    mutate(Comparison = "HP vs SSc-ILD") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5),  
  deg_table[["IPFvsIPAF"]] %>% 
    mutate(Comparison = "IPF vs IPAF") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5),  
  deg_table[["SScILDvsIPAF"]] %>% 
    mutate(Comparison = "SSc-ILD vs IPAF") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5),  
  deg_table[["IPFvsRA"]] %>% 
    mutate(Comparison = "IPF vs RA-ILD") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5),  
  deg_table[["HPvsRA"]] %>% 
    mutate(Comparison = "HP vs RA-ILD") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5),  
  deg_table[["SScILDvsRA"]] %>% 
    mutate(Comparison = "SSc-ILD vs RA-ILD") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=5)
)

colnames(ild_deg) <- c("Comparison","Ensembl ID","Gene","(ref:logfc)", "(ref:p-it)-value", "FDR")

kableExtra::kbl(
  ild_deg,
  longtable = F,
  format = 'latex',
  booktabs = T, 
  #linesep = "",
  align = "llrcc",
  caption="(ref:ILDdeg-cap)") %>%
  kableExtra::kable_styling(full_width = TRUE) %>%  # table spans across left to right margins
  column_spec(c(1), width="1.3in") %>%
  column_spec(c(2), width="1.2in") %>%
  column_spec(c(3), width="0.7in") %>%
  column_spec(c(4), width="0.6in") %>%
  column_spec(c(5), width="0.6in") %>%
  column_spec(c(6), width="0.6in") %>%
  #kableExtra::kable_styling(latex_options = "scale_down") #%>% 
  kableExtra::kable_styling(latex_options = c("hold_position"), font_size = 8) # ensures the table isn't "floated" and pins it to the current location
```

(ref:degrad-cap) \textbf{List of top 10 differentially expressed genes between radiological patterns.} Results shown are adjusted for race. 

```{r degrad, eval = T, echo = F, warning = FALSE, message=FALSE, out.extra='', fig.scap='DEGs: IPF vs SSc-ILD', fig.align='center'}
rad_table <- read_excel_allsheets(here::here('Tables', 'BloodRNAseq', "DEG_radiology_race_240529.xlsx"))

rad_deg <- rbind(
   rad_table[["UIPvsHP"]] %>% 
    mutate(Comparison = "UIP vs fHP") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=10),
  rad_table[["UIPvsNSIP"]] %>% 
    mutate(Comparison = "UIP vs NSIP") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=10),  
  rad_table[["HPvsNSIP"]] %>% 
    mutate(Comparison = "fHP vs NSIP") %>% 
    mutate_at(c("logFC","adj.P.Val"), round_to_sigfigs, sigfigs = 3) %>%
    mutate(P.Value=formatC(P.Value, format = "e", digits=2)) %>%
    dplyr::select(Comparison, ensembl_gene_id, hgnc_symbol, logFC, P.Value, adj.P.Val) %>% head(n=10) 
)

colnames(rad_deg) <- c("Comparison","Ensembl ID","Gene","(ref:logfc)", "(ref:p-it)-value", "FDR")

kableExtra::kbl(
  rad_deg,
  longtable = F,
  format = 'latex',
  booktabs = T, 
  linesep = "",
  align = "lllrcc",
  caption="(ref:degrad-cap)") %>%
  kableExtra::kable_styling(full_width = TRUE) %>%  # table spans across left to right margins
  column_spec(c(1), width="1.3in") %>%
  column_spec(c(2), width="1.2in") %>%
  column_spec(c(3), width="0.7in") %>%
  column_spec(c(4), width="0.6in") %>%
  column_spec(c(5), width="0.6in") %>%
  column_spec(c(6), width="0.6in") %>%
  #kableExtra::kable_styling(latex_options = "scale_down") #%>% 
  kableExtra::kable_styling(latex_options = c("hold_position"), font_size = 8) # ensures the table isn't "floated" and pins it to the current location
```

(ref:ppfdeg-cap) \textbf{Differentially expressed genes in PPF.} Volcano plots of (A) PPF, (B) relative decline in FVC% predicted of >10%, (C) radiological progression, and (D) relative decline in FVC% predicted of >5% after 2 years. (E-G) Boxplots of significant DEGs associated with various PPF criteria. 

```{r rnaseqPCA, eval = T, out.width='0.9\\linewidth',echo=FALSE, fig.cap="(ref:ppfdeg-cap)", fig.scap = "Correlation of PCs with metadata (blood RNA-seq)", out.extra='', fig.align='center'}
knitr::include_graphics("./Figures/BloodRNAseq/ppf_deg_v1.pdf")
```

- ppf, ild, ipf, sscild, nsip DEG table
- rad progression, ild, ipf, sscild, nsip DEG table
- ipf volcano plot
- 