---
output:
  bookdown::pdf_book:
    toc : no
    keep_tex: no
    number_sections: yes
---

<!-- This section is mandatory! -->

Given the scarcity of disease biomarkers in ILD, investigations into molecular biomarkers of ILD (particularly circulating biomarkers) are an ongoing area of interest [@clynick_biomarker_2022; @axelsson_proteomic_2022; @bowman_proteomic_2022]. A 52-gene signature derived from PBMC transcriptomic profiling has been shown to be predictive of transplant-free survival in patients with IPF [@herazo-maya_peripheral_2013;@herazo-maya_validating_2017], and transcriptomic profiling has also identified genes associated with decline in lung function in the ILD subtypes IPF and HP [@yang_peripheral_2012; @huang_blood_2021; @fernandez_perez_prognostic_2022]. 

While it is evident that the peripheral transcriptome is associated with prognosis in ILD, no investigations have been performed regarding its diagnostic potential. The objective of this study was to evaluate the ability of the whole blood transcriptome in discriminating between ILD subtypes. In this pilot study, we profiled whole blood RNA obtained from patients with ILD (n=59) using a previously published NanoString assay designed to discriminate between the early- and late-phase asthmatic response [@singh_novel_2018]. Given that fibrosis is a T\textsubscript{H}2-driven process like asthma [@spagnolo_role_2022], we reasoned that the transcripts profiled in the assay might also differentiate between other T\textsubscript{H}2-like disease subtypes.  

## Methods

### Study population and sample processing

Patients were diagnosed with fibrotic ILD (IPF, n=22; HP, n=14; SSc-ILD, n=20; IPAF, n=3) in accordance with diagnostic guidelines and prospectively recruited with written informed consent (ethics protocol H09-00748) between 2012 and 2018 at St. Paul’s Hospital (Vancouver, BC, Canada). At time of sampling, patients were either undergoing anti-fibrotic (pirfenidone, nintedanib), anti-inflammatory (prednisone, mycophenolate, N-acetylcysteine), or no pharmacotherapy. Whole blood samples were collected in PAXgene Blood RNA Tubes (BD Biosciences, Mississauga, ON, Canada) and extracted for RNA using the PAXgene Blood miRNA Kit (PreAnalytiX, Hombrechtikon, Switzerland). RNA quality was confirmed by the RNA 6000 Nano Kit (Agilent, Santa Clara, California, USA) via RNA integrity numbers obtained from Bioanalyzer analysis. Blood transcript expression was profiled using 100 ng of purified RNA with a custom NanoString nCounter Elements assay (NanoString Technologies, Seattle, WA, USA) that measured 166 transcripts [@singh_novel_2018]. Samples were randomized across six NanoString cartridges (12 samples per cartridge), with remaining assay slots used for sample replicates to assess data reproducibility.

### In situ hybridization

Custom RNAscope™ probes for KLRF1 (276-375 of NM_016523.1), LTK (2419-2518 of NM_001135685.1), and VCAN (transcript variant 3; 1356-1455 of uc003kij.3) were developed by Advanced Cell Diagnostics (ACD, Newark, California). Tissue microarray (TMA) sections (4$\mu$m-thick) were obtained from a formalin-fixed and paraffin-embedded TMA block containing cores from donor lungs of patients with IPF, HP, and SSc-ILD collected at McMaster University. In situ hybridization (ISH) was performed using a Leica BOND RX autostainer with associated Leica reagent kits (Concord, Ontario, Canada) on TMA sections. Slides were scanned at 40X magnification with an Olympus VS120-L100 Virtual Slide System for image capture and a Leica Aperio ScanScope AT2 for quantification. Image analysis was performed using QuPath (v0.4.3), and probe quantification was determined using H-scores as indicated by manufacturer protocols.

### Data analysis

NanoString data underwent quality assessment by examining field of view (FOV) ratio (number of FOV images successfully captured), binding density (image saturation), linearity of positive controls, and limit of detection via negative controls. Data were filtered for lowly abundant features, then normalized using normalization factors derived from the geometric mean of 10 housekeeping transcripts. Differential expression analysis of transcripts was performed using the linear models for microarray (limma) R package (v3.54.2), while biomarker panels were developed using sparse partial least squares discriminant analysis (sPLSDA), elastic net, and random forest algorithms from the ‘mixOmics’ (v6.22.0) and ‘tidymodels’ (v1.0.0) R packages. Image analysis was performed using QuPath (v0.4.3) by automatically identifying individual cells and setting probe-specific thresholds based on signal strength. Cells without a visible nucleus were excluded from analysis. H-scores were determined by grouping cells into 5 bins based on the numbers of dots per cell (0, 1-3, 4-9, 10-15, >15) and multiplying the total percentage of cells in each bin by 0 to 4. To test for statistical significance of H-scores, a linear mixed-effects model (‘lme4’ v1.1-32) was fitted with the formula H-Score ~ Diagnosis + (1|Patient) and pairwise comparisons between diagnoses to determine statistical significance with Tukey’s post-hoc adjustment were performed using the ‘emmeans’ (v1.8.5) R package. Fisher’s exact test was used to test for differences in demographic count variables between diagnoses, while the F-test was used to test for differences in demographic continuous variables between diagnoses. All analysis was performed in the R (version 4.2.3) statistical computing environment. 

## Results

We profiled the expression of 166 RNA transcripts in a cohort of patients with IPF, systemic sclerosis-associated ILD (SSc-ILD), HP, or interstitial pneumonia with autoimmune features (IPAF) (Table \@ref(tab:bloodpx)). No significant differences were identified between subtypes for race, percent predicted forced vital capacity (FVC\%) or diffusing capacity of the lung for carbon monoxide (DLCO\%), smoking status, or treatment status, but patients with SSc-ILD and IPAF were younger at time of sampling and patients with IPF were predominantly male which is in line with previously reported cohorts [@zaman_differences_2020]. 

\captionsetup{width=6.5in} <!-- makes table caption stays in margins -->

(ref:p-it) \textit{p}

(ref:bloodpx-cap) \textbf{Summary table of clinical demographics of patients probed for whole blood RNA expression.} 

```{r bloodpx, eval = T, echo = F, warning = FALSE, message=FALSE, out.extra='', fig.align='center'}
bloodpx <- readr::read_csv(here::here('Tables', 'BloodPilot', 'Table 1 Blood Px.csv'))
colnames(bloodpx)[1] <- ""
kableExtra::kbl(
  bloodpx,
  longtable = F,
  format = 'latex',
  booktabs = T, 
  linesep = "",
  align = "lccccc",
  caption="(ref:bloodpx-cap)",
  caption.short = "ILD blood pilot study cohort") %>%
  kableExtra::kable_styling(full_width = TRUE) %>%  # table spans across left to right margins
  column_spec(1, width="1.3in") %>%
  column_spec(2, width="0.9in") %>%
  column_spec(3, width="1.0in") %>%
  column_spec(4, width="0.9in") %>%
  column_spec(5, width="0.9in") %>%
  column_spec(6, width="0.75in") %>%
  row_spec(c(5,10),hline_after=TRUE) %>%
  #row_spec(37:38, bold=T) %>%
  #kableExtra::kable_styling(latex_options = "scale_down") #%>% 
  kableExtra::kable_styling(latex_options = c("hold_position"), font_size = 9) # ensures the table isn't "floated" and pins it to the current location
```

To identify variables that might affect RNA expression, we performed a principal component (PC) analysis and correlated the PCs with demographic variables (\@ref(fig:pilotPCA)). After adjusting for race and treatment status, we identified 1 differentially expressed transcript at an FDR of 0.05 when comparing IPF to SSc-ILD (\textit{KLRF1}, FDR = 0.03) (Table \@ref(tab:pilotdiffexp), Figure \@ref(fig:pilotexpr)D). While no transcripts were significantly associated with FVC%, we identified 13 transcripts that were significantly associated with DLCO% (Table \@ref(tab:pilotpftdeg)). Next, we examined the utility of RNA profiling in developing biomarker panels to differentiate between ILD subtypes through the use of three learning algorithms (sPLSDA, elastic net, and random forest). The best-performing models using 5-fold cross-validation repeated ten times were IPF vs SSc-ILD (sPLSDA) and IPF vs SSc-ILD (elastic net), which yielded respective accuracies of 0.74 ± 0.01 and 0.80 ± 0.02 (Table \@ref(tab:biomarkermodel)). 

\newpage
\captionsetup{width=6.5in} <!-- makes table caption stays in margins -->
\begin{singlespace}
(ref:ipfnonipf) \textbf{IPF vs non-IPF}
(ref:ipfvshp) \textbf{IPF vs HP}
(ref:ipfvsssc) \textbf{IPF vs SSc-ILD}
(ref:hpvsssc) \textbf{HP vs SSc-ILD}
(ref:logfc) log\textsubscript{2}FC

(ref:pilotdiffexp-cap) \textbf{List of transcripts analyzed for differential expression between ILD subtypes.} Results shown are adjusted for using treatment status and race. 

```{r pilotdiffexp, eval = T, echo = F, warning = FALSE, message=FALSE, out.extra='', fig.scap='ILD pilot DEGs', fig.align='center'}
deg_table <- readr::read_csv(here::here('Tables', 'BloodPilot', 'diffexp.csv'))
deg_table <- deg_table %>% mutate(logFC=signif(logFC, digits=2)) %>% mutate(P.Value = round(P.Value, digits=2)) %>% mutate(adj.P.Val = round(adj.P.Val, digits=2)) %>%
  rename(`(ref:logfc)`=logFC) %>% rename(`(ref:p-it)-value`=P.Value) %>% rename(FDR=adj.P.Val)
kableExtra::kbl(
  deg_table,
  longtable = F,
  format = 'latex',
  booktabs = T, 
  linesep = "",
  align = "lccc",
  caption="(ref:pilotdiffexp-cap)",
  caption.short = "Blood pilot study DEGs") %>%
  kableExtra::kable_styling(full_width = TRUE) %>%  # table spans across left to right margins
  column_spec(1, width="1.3in") %>%
  column_spec(2, width="0.7in") %>%
  column_spec(3, width="0.7in") %>%
  column_spec(4, width="0.7in") %>%
  #row_spec(c(6,12,18),hline_after=TRUE) %>%
  #row_spec(37:38, bold=T) %>%
  #kableExtra::kable_styling(latex_options = "scale_down") #%>% 
  kableExtra::kable_styling(latex_options = c("hold_position"), font_size = 9) # ensures the table isn't "floated" and pins it to the current location
```

\end{singlespace}

To investigate the lung expression of the three most highly weighted genes (\textit{KLRF1}, \textit{LTK}, \textit{VCAN}) identified by our blood biomarker panels when comparing IPF vs non-IPF ILD (Table \@ref(tab:biomarkerweight)), we used ISH on TMA sections consisting of twelve cores from five patients with IPF, ten cores from three patients with HP, and four cores from two patients with SSc-ILD (Figure \@ref(fig:pilotexpr)A-C), all of which showed no between-group differences in demographic or clinical variables (Table \@ref(tab:lungpx)). Transcript abundance was determined semi-quantitatively for each marker per core by calculating H-scores, which is a weighted total of cells expressing the probe. Using a linear mixed-effects analysis, the H-score of \textit{KLRF1} was higher in SSc-ILD compared to IPF (\textit{p}=0.02) and HP (\textit{p}=0.01), while \textit{LTK} was lower in IPF compared to SSc-ILD (\textit{p}=0.03) (Figure \@ref(fig:pilotexpr)). 

\captionsetup{width=6.5in} <!-- makes table caption stays in margins -->

(ref:lungpx-cap) \textbf{Summary table of clinical demographics of patients probed for \textit{LTK}, \textit{KLRF1}, and \textit{VCAN} expression using in situ hybrdization in lung tissue microarray sections.} 

```{r lungpx, eval = T, echo = F, warning = FALSE, message=FALSE, out.extra='', fig.scap='ILD pilot study cohort - lung', fig.align='center'}
lungpx <- readr::read_csv(here::here('Tables', 'BloodPilot', 'Table 2 Lung Px.csv'))
colnames(lungpx)[1] <- ""

kableExtra::kbl(
  lungpx,
  longtable = F,
  format = 'latex',
  booktabs = T, 
  linesep = "",
  align = "lccc",
  caption="(ref:lungpx-cap)",
  caption.short = "ILD lung pilot study cohort") %>%
  kableExtra::kable_styling(full_width = TRUE) %>%  # table spans across left to right margins
  column_spec(1, width="1.3in") %>%
  column_spec(2, width="1.0in") %>%
  column_spec(3, width="1.0in") %>%
  column_spec(4, width="1.0in") %>%
  column_spec(5, width="1.0in") %>%
  #row_spec(37:38, bold=T) %>%
  #kableExtra::kable_styling(latex_options = "scale_down") #%>% 
  kableExtra::kable_styling(latex_options = c("hold_position"), font_size = 9) # ensures the table isn't "floated" and pins it to the current location
```

(ref:pilotexpr-cap) \textbf{Expression of \textit{KLRF1}, \textit{LTK}, \textit{VCAN} in whole blood (log2 RNA expression) or lung tissue microarray sections (H-Score) containing samples from patients with ILD subtypes.} Tissue microarray sections of lung samples from patients with (A) idiopathic pulmonary fibrosis (IPF), (B) hypersensitivity pneumonitis (HP), and (C) systemic sclerosis-associated interstitial lung disease (SSc-ILD). (D) Whole blood (log2 RNA expression) and lung (H-Score) expression of \textit{KLRF1}, \textit{LTK}, and \textit{VCAN}. Statistical significance of blood RNA expression was corrected using false discovery rate (FDR), with significance indicated by FDR<0.05 (\*\*\*). Tukey’s post-hoc adjustment was used to determine statistical significance of H-scores (\* denotes \textit{p}<0.05). 

```{r pilotexpr, eval = T, out.width='0.9\\linewidth',echo=FALSE, fig.cap="(ref:pilotexpr-cap)", fig.scap = "Blood and lung expression of LTK, KLRF1, VCAN", out.extra='', fig.align='center'}
knitr::include_graphics("./Figures/BloodPilot/ILD_biomarkers_combined_thesis_v1.png")
```

## Discussion

Although previous studies have investigated the peripheral transcriptome in relation to prognosis [@herazo-maya_peripheral_2013;@yang_peripheral_2012;@fernandez_perez_prognostic_2022], no study has looked at its diagnostic potential. In this pilot study, we demonstrate that the peripheral blood RNA can differentiate between ILD subtypes, in particular IPF and SSc-ILD, through differential expression and learning algorithms utilizing gene expression data obtained from a custom NanoString assay. The blood gene expression data show association with DLCO\% predicted but not FVC\% predicted, which was previously noted when examining differential expression of the peripheral transcriptome in mild and severe IPF [@yang_peripheral_2012]. Furthermore, we identify \textit{KLRF1} expression as a significant in the differentiation of IPF and SSc-ILD in both whole blood and lung, and down-regulation of \textit{LTK} as being associated with IPF.

Killer cell lectin like receptor F1 (KLRF1, also known as NKp80) is expressed in T cells, monocytes, and CD56bright natural killer (NK cells), and is a marker of NK cell maturity [@roda-navarro_human_2000; @freud_nkp80_2016; @mukherjee_klrf1_2023], inducing cytotoxic activity and calcium mobilization [@vitale_identification_2001]. The dysfunction of NK cells in systemic sclerosis and autoimmunity has been well-documented [@horikawa_abnormal_2005; @benyamine_natural_2018; @liu_nk_2021], and differences in their abundance have been observed between ILD subtypes [@esposito_natural_2005;@bergantini_prognostic_2021]. We found that \textit{KLRF1} expression was associated with lower DLCO%, and its expression was lower in the peripheral blood of patients with SSc-ILD compared to IPF and HP yet the opposite effect was observed when probing lung expression. Previous reports have identified a reduction in abundance and function of NK cells in IPF lungs concurrent with an increase in abundance in blood compared to healthy controls [@cruz_reduced_2021]. Single-cell RNA-sequencing data available via the IPF Cell Atlas [@neumark_idiopathic_2020] suggests higher \textit{KLRF1} expression in SSc-ILD T/NK subsets compared to other ILDs. Given the disparity between single-cell and bulk RNA data, further investigations are required to delineate the mechanisms surrounding NK cell recruitment to the ILD lung and its role in ILD pathogenesis. 

\textit{LTK} and \textit{VCAN} were differentially expressed at a nominal p-value of 0.05 between IPF and non-IPF ILDs (but not at FDR<0.05), and were two highly weighted coefficients in our elastic net and sPLSDA biomarker panels. \textit{LTK} is highly expressed in plasmacytoid dendritic cells (pDCs) (Human Protein Atlas, IPF Cell Atlas), and decreased circulating pDCs have been reported in IPF patients [@galati_circulating_2020], which is consistent with the decreased \textit{LTK} expression observed in this study in blood and lung. \textit{VCAN} encodes for versican, an ECM proteoglycan that is localized to fibroblastic foci in the lung, myofibroblasts, and monocytes [@neumark_idiopathic_2020;@bensadoun_proteoglycan_1996]. We were unable to identify a statistically significant increase in \textit{VCAN} expression in IPF compared to HP or SSc-ILD lung and blood samples; however, given its detectable expression in both lung and peripheral blood, and prior studies showing circulating levels of versican degradation products are associated with increased mortality in idiopathic ILDs [@sand_serological_2018], further studies into its gene expression as a prognostic biomarker may be warranted.

In this pilot study, we report the first between-subtype comparison of the peripheral ILD transcriptome using a discrete set of genes, which validates performing more extensive genome-wide analysis of peripheral ILD subtype-specific profiles and correlations of identified biomarkers in lung tissue. In particular, molecules detected in the blood that are associated with immunity and fibrosis are detectable in the lung, thereby suggesting a test based on peripheral transcriptomics is feasible for diagnostic biomarkers with biological relevance, similar to prognostic markers in Herazo-Maya et al. [@herazo-maya_validating_2017]. Using the data from this preliminary pilot study, we were able to secure funding for two grants (BC Lung, Genome BC) that created the basis for the following two chapters.
