---
output:
  bookdown::pdf_book:
    toc : no
    keep_tex: no
    number_sections: yes
---

<!-- This section is mandatory! -->

The results from the previous two chapters highlight the link between transcriptomic profiles, mechanisms of ILD, and clinical variables. Given the heterogeneity within ILD subtypes, analysis of the peripheral transcriptome should consider radiological pattern, progressive phenotype, and other related clinical factors [@ryerson_lumpers_2019]. In this chapter, we will explore the peripheral transcriptome using RNA-seq data from whole blood samples of patients with ILD. 

## Methods

### Study population and sample processing

As part of the Canadian Registry for Pulmonary Fibrosis (CARE-PF), patients with ILD were prospectively recruited with written informed consent between 2012 and 2019 at the Pacific Lung Health Centre located at St. Paul's Hospital (Vancouver, BC, Canada). Demographic (age, race, sex) and clinical data (FVC%, D\textsubscript{LCO}% predicted) were collected at time of sampling (Table \@ref(tab:bloodpxrnaseq). ILD diagnoses were determined by consensus agreement by a MDD panel consisting of at least one pulmonologist, radiologist, and pathologist, and independently re-evaluated to confirm diagnoses and other clinical features (including HRCT patterns) between January 2021 and March 2022 [@marinescu_integration_2023]. Patients were defined as having PPF on the basis of the INBUILD criteria (see: [Introduction - Progressive pulmonary fibrosis](#ppf)). 

At time of recruitment, patient whole blood samples were collected in PAXgene Blood RNA Tubes (BD Biosciences, Mississauga, ON, Canada). We selected samples to examine differences between patients with ILD that experience PPF (n=39) and those that did not (n=57) with a balanced representation of ILD subtypes, and included samples if patients were not undergoing active pharmalogical treatment at time of blood sampling. PAXgene-preserved whole blood samples were extracted for RNA using the PAXgene Blood miRNA Kit (PreAnalytiX, Hombrechtikon, Switzerland), and quality control was performed using the Agilent 2100 Bioanalyzer (Agilent, Santa Clara, California, USA). Sample library preparation was performed using the Illumina Stranded mRNA Prep kit (Illumina, San Diego, California, USA), then sequenced on the Illumina NextSeq 2000 sequencing system with paired end 59bp x 59bp reads. Due to hemoglobin-associated genes comprising a majority of reads in whole blood transcriptome sequencing [@shin_variation_2014], each sample was sequenced to a depth a 50 million reads. Sequencing data quality check was performed using fastqc (v0.11.9), and sequence alignment to the human reference genome (GENCODE v41) was performed using STAR (v2.7.10a). Transcript abundance was estimated using featureCounts (v2.0.3). Hemoglobin genes (\textit{HBA1}, \textit{HBA2}, \textit{HBB}) were removed from the final count matrix due to their overrepresentation. 

\begin{landscape}

\captionsetup{width=8.5in}

(ref:race) \textbf{Race, n (\%)}

(ref:bloodpxrnaseq-cap) \textbf{Summary table of demographics of patients with ILD examined for whole blood RNA expression.} Data are shown as either n(\%) or mean Â± SD (calculated for all patients within diagnoses unless otherwise indicated). \textit{P}-values were determined using one-way analysis of variance (ANOVA) for continuous variables and Fisher's exact text for categorical variables. Variables relating to progression were determined within a 2-year period. 

```{r bloodpxrnaseq, eval = T, echo = F, warning = FALSE, message=FALSE, out.extra='', fig.scap='ILD blood RNA-seq study cohort', fig.align='center'}
bloodpxrnaseq <- readr::read_csv(here::here('Tables', 'BloodRNAseq', 'px_summary.csv'))
colnames(bloodpxrnaseq)[1] <- ""
kableExtra::kbl(
  bloodpxrnaseq,
  longtable = F,
  format = 'latex',
  booktabs = T, 
  linesep = "",
  align = "lccccccl",
  caption="(ref:bloodpxrnaseq-cap)") %>%
  kableExtra::kable_styling(full_width = TRUE) %>%  # table spans across left to right margins
  column_spec(1, width="1.5in") %>%
  column_spec(2, width="1.05in") %>% #SSc
  column_spec(3, width="1.05in") %>% #IPF
  column_spec(4, width="0.97in") %>% #HP
  column_spec(5, width="0.97in") %>% #RA
  column_spec(6, width="0.95in") %>% # IPAF
  column_spec(7, width="1.3in") %>% # Unc
  column_spec(8, width="0.7in") %>% # p
  row_spec(c(3,8,11),hline_after=TRUE) %>%
  row_spec(c(1,9,12), bold=T) %>%
  #kableExtra::kable_styling(latex_options = "scale_down") #%>% 
  kableExtra::kable_styling(latex_options = c("hold_position"), font_size = 8) # ensures the table isn't "floated" and pins it to the current location
```

\end{landscape}

### Data analysis

Differential expression analysis was performed using the 'edgeR' (v4.0.16) and 'limma' (v3.58.1) R packages. Lowly expressed genes (less than 10 counts and based on CPM as per recommended limma protocols) were filtered out if they were not expressed in at least four samples (chosen based on the sample size of the smallest ILD subtype, IPAF). The total number of genes in the final count matrix was 24,990. Genes were TMM-normalized and examined for outliers using multidimensional scaling (MDS) and PCA plots. Design matrices containing variables of interest were used to test for differential expression, with a term added into the formula to control for race as a confounding variable (Figure \@ref(fig:rnaseqPCA)). 

Survival analysis was performed using the elastic net implementation of Cox proportional hazards (Cox-PH) models in the 'glmnet' package (v4.1-8). Repeated 10-fold cross-validation was performed on all (100x repeat) or stratifications (50x repeat) of the centred and scaled TMM-normalized RNA-seq data. Using the C-index as a performance metric, the repeat with the C-index closest to the mean C-index of all repeats was selected as the final model. Validation datasets GSE28042 [@herazo-maya_peripheral_2013] and GSE93606 [@molyneaux_host-microbial_2017] were downloaded from GEO. As there were genes from the whole blood RNA-seq matrix missing from the validation datasets and vice versa, sparse matrices of each validation dataset were created for compatibility with the glmnet 'predict' function to account for missing variables. 

## Results

### The peripheral transcriptome is distinct between ILD subtypes and radiological patterns

We first examined differences in the peripheral transcriptome of specific ILD subtypes against each other. After controlling for race as a confounding variable (Figure \@ref(fig:rnaseqPCA)), IPF and SSc-ILD were the most transcriptionally distinct subtypes with 165 DEGs at an FDR<0.05, a majority of which were associated with interferon signaling (Figure \@ref(fig:degild)A,C-D, Table \@ref(tab:ipfvsssc), Table \@ref(tab:ipfsscpathway)). No DEGs were identified between IPF and HP (Figure \@ref(fig:degild)B), but two DEGs at FDR<0.05 were identified between IPF and IPAF and 710 between SSc-ILD and IPAF, though these comparisons were limited by the small number of IPAF samples (n=4) (Table \@ref(tab:ILDdeg)). We further investigated DEGs between radiological subtypes, and found that fibrotic HP patterns were distinct from NSIP patterns with 16 DEGs as FDR<0.05 (Figure \@ref(fig:raddeg)); similar genes were upregulated when compared to UIP, though these were not significant (Table \@ref(tab:degrad)).

(ref:degild-cap) \textbf{SSc-ILD is transcriptionally distinct.} Volcano plots of differentially expressed genes between (A) IPF and SSc-ILD and (B) IPF and HP. (C) Expression of interferon- and cytotoxic-associated genes in ILD subtypes. Statistical significance of blood RNA expression was corrected using false discovery rate (FDR), with significance indicated by FDR<0.01 (\*\*\*) and FDR<0.05 (\*\*). (D) Gene-concept plot of selected KEGG and Reactome pathways associated with up- and down-regulated SSc-ILD genes when compared to IPF. 

```{r degild, eval = T, out.width='1.0\\linewidth',echo=FALSE, fig.cap="(ref:degild-cap)", fig.scap = "SSc-ILD transcriptome", out.extra='', fig.align='center'}
knitr::include_graphics("./Figures/BloodRNAseq/Fig1_Differential_Expression_v1.pdf")
```

(ref:raddeg-cap) \textbf{Radiological patterns have distinct peripheral transcriptomes.} Volcano plots of differentially expressed genes between (A) fHP and NSIP and (B) fHP and UIP. (C) Expression of top DEGs in fHP. Satistical significance of blood RNA expression was corrected using false discovery rate (FDR), with significance indicated by FDR<0.01 (\*\*\*), FDR<0.05 (\*\*), and FDR<0.10(\*).

```{r raddeg, eval = T, out.width='1.0\\linewidth',echo=FALSE, fig.cap="(ref:raddeg-cap)", fig.scap = "Radiological transcriptome", out.extra='', fig.align='center'}
knitr::include_graphics("./Figures/BloodRNAseq/radiology_deg_v1.pdf")
```

### Genes are associated with PPF at baseline in SSc-ILD and NSIP patients

To investigate the association of the peripheral transcriptome at baseline with future development of PPF, we used the INBUILD criteria to classify whether or not patients had a PPF phenotype after 2 years. We did not identify any DEGs associated with PPF status in all ILD subtypes, although we identified a small number of genes significantly associated with INBUILD criteria such as relative decline in FVC of >5%, >10%, and radiological progression (Figure \@ref(fig:ppfdeg)). When the analysis was stratified for SSc-ILD and NSIP, we identified 

- ssc nsip
- ipf: only one deg
- nothing found for any other stratified analysis 

### Peripheral gene signatures are associated with transplant-free survival

Cox-PH models associate covariates with time that passes before a pre-defined event; in this analysis, we wanted to identify multivariate gene signatures associated with transplant-free survival. 

## Discussion

- limitations regarding survival: 15/27 IPF patients censored, which is 15/25 of the total censored 
- ssc-ild vs other ild is not difficult diagnostically
- fHP shows neutrophil signatures (LCN2, MMP8, CEACAM8); KREMEN1 blocks Wnt signaling






