---
output:
  bookdown::pdf_book:
    toc : no
    keep_tex: no
    number_sections: yes
---

<!-- This section is mandatory! -->

It is evident that the peripheral transcriptome contains diagnostic and prognostic markers of ILD. With the advent of scRNA-seq, increased resolution of transcriptomic profiling will allow for the discovery of cell-specific biomarkers. To date, scRNA-seq studies of ILD have mainly focused on the lung cellular landscape. In human IPF lung, scRNA-seq profiling has revealed transcriptomic changes consistent with the hypothesis that lung epithelial cells undergo significant differentiation in IPF, with two studies finding specific clusters of cells expressing transcripts consistent with a possible AT2 to AT1 differentiation state [@xu_single-cell_2016; @habermann_single-cell_2020]. In both human ILD lung and a bleomycin mouse model of lung fibrosis, distinct populations of profibrotic macrophages have been identified [@aran_reference-based_2019; @reyfman_single-cell_2019]. Despite these investigations, a key question remains: what is the source of the novel profibrotic myofibroblasts and macrophages in ILD? It is likely that circulating progenitor cells play a key role. Within the bleomycin mouse model, lung fibrosis is driven by alveolar macrophages derived from circulating monocytes that can persist for at least a year upon resolution of fibrosis [@misharin_monocyte-derived_2017]. A recent scRNA-seq study identified an ACTA2-positive myofibroblast population specific to IPF (when compared with COPD and healthy controls), but gene expression profiling suggested that it did not arise from resident or activated lung fibroblasts [@adams_single-cell_2020]. Therefore, in order to identify potential biomarkers of ILD, blood is an extremely promising sampling medium with informative biology and minimal risk to patients. This chapter covers a preliminary cohort of scRNA-seq samples derived from patients with IPF and HP. 

## Methods

### Study population and sample processing

Patients with suspected ILD were prospectively recruited (UBC ethics H21-00291) at the Pacific Lung Health Centre (St. Paul's Hospital, Vancouver, BC, Canada) from 2021-2023 in conjunction with the ILD Biomarkers Registry (UBC ethics H23-01364). Reviews of existing patient data (HRCT scan, history, referral notes) were conducted prior to scheduled appointments and assessed for likelihood of an ILD diagnosis. If pre-appointment data was not sufficient to assess a likely diagnosis, consultation with physician notes or physicians after the appointment was performed to determine whether a sample would be processed or not. Patients with IPF, HP, IPAF, and unclassifiable ILD were prioritized for sample collection, while those with sarcoidosis, radiation-induced ILD, or occupational lung exposures (e.g. asbestosis) were not collected given their differences in disease progression compared to other fibrotic ILDs [@cottin_presentation_2018]. Patients on anti-inflammatory or anti-fibrotic treatment were omitted due to possible confounding effects on transcriptional profiling, and samples were only taken from patients who were not yet diagnosed and likely to be early in their disease course. 

Blood samples were collected in EDTA tubes and PBMCs were isolated with density gradient centrifugation using SepMate\texttrademark-15 tubes and Lymphoprep\texttrademark according to recommended protocols (STEMCELL Technologies, Vancouver, BC, Canada). At time of blood sampling, a complete blood count (CBC) was also collected using the Siemens Advia\textregistered 2120i (Siemens Canada, Oakville, Ontario, Canada). Isolated PBMCs were immediately transported to the sequencing facility at the UBC School of Biomedical Engineering, confirmed to be >90% viable with trypan blue staining, and processed using the 10X Genomics Chromium Single Cell 3' Kit (10X Genomics, Pleasanton, California, USA) according to manufacturer-recommended protocols. Briefly, the 10X Chromium Controller captures cells in GEMs (gel beads in emulsion), which contain a single cell and sufficient reagents for library construction via reverse transcription. Sequencing was performed using an Illumina NextSeq 2000 for 100 cycles with a targeted depth of ~30,000 reads per cell, and sequencing results were processed, assessed for data quality, and exported using Cell Ranger v6 (10X Genomics). 

(ref:scrnaseqpx-cap) \textbf{Summary table of demographics of patients with ILD examined for whole blood RNA expression.} Data are shown as either n(\%) or mean Â± SD (calculated for all patients within diagnoses). \textit{P}-values were determined using one-way analysis of variance (ANOVA) for age and FVC, and Fisher's exact text for sex.

```{r scrnaseqpx, eval = T, echo = F, warning = FALSE, message=FALSE, out.extra='', fig.scap='ILD PBMC scRNA-seq study cohort', fig.align='center'}
bloodpxrnaseq <- readr::read_csv(here::here('Tables', 'scRNAseq', 'px_summary.csv'))
colnames(bloodpxrnaseq)[1] <- ""
kableExtra::kbl(
  bloodpxrnaseq,
  longtable = F,
  format = 'latex',
  booktabs = T, 
  linesep = "",
  align = "lcccc",
  caption="(ref:scrnaseqpx-cap)") %>%
  kableExtra::kable_styling(full_width = FALSE) %>%  # table spans across left to right margins
  column_spec(1, width="1.75in") %>%
  column_spec(2, width="1.05in") %>% 
  column_spec(3, width="1.05in") %>% 
  column_spec(3, width="1.0in") %>%
  #kableExtra::kable_styling(latex_options = "scale_down") #%>% 
  kableExtra::kable_styling(latex_options = c("hold_position"), font_size = 8) # ensures the table isn't "floated" and pins it to the current location
```

### Data analysis

Single-cell samples were individually processed to filter out ambient RNA using 'SoupX' (v1.6.2), cells with >15\% mitochondrial reads, cells undergoing proliferation using G2M and S phase scores, and doublets or multiplets using 'DoubletFinder' (v2.0.4). Data was integrated and clustered at 1.5 resolution using the R package 'Seurat' (v4.0). Annotations for clusters were determined using the Monaco PBMC reference dataset with the 'celldex' R package (v1.12.0) [@monaco_rna-seq_2019; @aran_reference-based_2019]. Differential expression analysis of normalized RNA data was performed using the Seurat implementation of the MAST (Model-based Analysis of Single-cell Transcriptomics) framework [@finak_mast_2015]. Given the difference in sample size between IPF and HP patients, comparisons were down sampled to the lower number of cells within each cluster. 

To test for cell abundances, a binomial generalized linear model was fit with the formula:

\begin{math}(N\textsubscript{cells in cluster}, N\textsubscript{cells not in cluster}) $\sim$ (Cluster \cdot Diagnosis) + (Cluster \cdot Capture Date) + (1|Subject)\end{math}

Comparisons were made using the 'emmeans' package (v1.10.1) and corrected using the Bonferroni method. A similar procedure was performed for CBC data without the term for capture date. 

## Results

### IPF and HP differ in cellular populations
- t helper expression of HLA... CD4 cells can expand into HLA-DR+ cells after Ag encounter (Tippalagama) and sepsis [Xu 2019]

We used scRNA-seq to profile PBMCs from blood samples of 7 IPF and 4 HP patients (Table \@ref(tab:scrnaseqpx)). After scRNA-seq sample processing and integration, we identified 27 cell clusters from 28,614 cells, all of which were previously identified in the reference dataset from Monaco \textit{et al.} (Figure \@ref(fig:clustermarkers)). We identified two subsets of classical monocytes, two subsets of effector CD8+ memory T cells, three subsets of naive CD4+ cells, and two subsets of T helper cells (Table \@ref(tab:clustermarkertab)). The classical monocyte subset C6 was characterized by expression of S100A8, S100A12, and PLBD1 compared to C12, while the classical monocyte C12 subset was characterized by increased  expression of AHNAK, AHR, ITGA4, HLA-DPB1, HLA-DPA1, HLA-DRB1, LGALS2, and F13A1 compared to C6. CD8+ memory T cells were separated into clusters C0 and C10, which expressed ACTB, ACTG1, CLIC1, CORO1A, WDR1 (C10) and IL7R, CXCR4 (C0), the latter of which is likely a more cytotoxic population [@micevic_il-7r_2023]. Naive CD4+ T cell subset C7 was characterized by increased CISH expression, previously identified as a marker of proinflammatory CD4+ T cells from older adults [@jin_cish_2023], while C9 had increased expression of TTN, which is involved in lymphocyte adhesion under flow [@toffali_isoform_2023]. The C3 Naive CD4+ T cell subset had increased ACTB and VASP expression but decreased ITGA4, ITGA6, and CDC14A expression, suggesting a possible motile phenotype [@waldman_enavasp_2022]. Expression of polarized T helper cell markers (GATA3, TBX21, RORC) was low and did not differentiate between both clusters, which expressed high levels of LTB, and differed by increased expression of the HLA subunits HLA-DRB1, HLA-DPA1, and HLA-DPB1 in C4, which may represent a subset of antigen-presenting CD4 T cells [@tippalagama_hla-dr_2021; @xu_dynamic_2020].

(ref:clustermarkers-cap) \textbf{scRNAseq cell clusters.} (A) Uniform manifold approximation and projection (UMAP) of scRNA-seq PBMC data with labelled clusters for each cell type as determined by mapping to a reference dataset. (B) Gene markers (columns) for each identified cluster (rows) indicated by numbers in A. Genes were selected using the 'top_markers' feature from the 'presto' R package, with a minimum AUC set to 0.5 and minimum percentage expression within the cluster of 20\%. 

```{r clustermarkers, eval = T, out.width='1\\linewidth', echo=FALSE, fig.cap="(ref:clustermarkers-cap)", fig.scap = "scRNA-seq clusters", out.extra=''}
knitr::include_graphics("./Figures/scRNAseq/uamp_cluster_v1.pdf")
```

Of the identified cell clusters, 'Classical monocytes C6', 'Intermediate monocytes', 'Low-density granulocytes', 'NK CD56 bright', and 'NK CD56 dim' were enriched in patients with HP, while patients with IPF were enriched in 'Central memory CD8 T cells', 'Naive CD8+ cells', 'Peripheral memory CD8 T cells', 'T helper cells C1', and 'T helper cells C4' (Figure \@ref(fig:forestplot)). The differences in scRNA-seq cluster representation were not observed in monocytes when examining CBCs taken at time of sampling, but IPF patients had significantly more lymphocytes and significantly fewer eosinophils when compared to HP patients, which corroborates the observed differences in T cell subset and low-density granulocytes from scRNA-seq clustering (Figure \@ref(fig:forestplotcbc)). 

(ref:forestplot-cap) Forest plot comparing abundance of cell clusters identified by scRNA-seq. Significantly different cell clusters (shown in italics) were determined by a binomial generalized linear model and corrected using the Bonferroni method. Dot sizes are scaled to indicate standard error (SE), with smaller SE corresponding to larger dots.

```{r forestplot, eval = T, out.width='1\\linewidth', echo=FALSE, fig.cap="(ref:forestplot-cap)", fig.scap = "scRNA-seq cell abundance", out.extra=''}
knitr::include_graphics("./Figures/scRNAseq/SCAILD_forestplot.pdf")
```

### IPF and HP cell types have differing transcriptional profiles

To investigate differentially expressed genes between IPF and HP within cell clusters, we used the MAST algorithm which avoids pseudoreplication bias in single-cell studies as outlined in [@zimmerman_practical_2021] (Table \@ref(tab:degscrnaseq)). Across multiple cell types, JUN was highly expressed in samples from patients with HP (Figure \@ref(fig:junumap)). In addition to being differentially abundant, NK cell subsets (NK CD56 bright, NK CD56 dim, natural killer T cells [NKT]) expressed increased MYOM2 in HP patients compared to IPF patients (Figure \@ref(fig:scrnaseqnk)). Moreover, in NK CD56 dim and NKT cells, we observed a downregulation of KLRC1 in IPF compared to HP samples, which was in contrast to an increase in KLRC2 expression. 

(ref:scrnaseqnk-cap) Differentially expressed genes between NK cell subsets of IPF and HP as shown by violin plots (A) and UMAP plots (B). Significance, as determined by the MAST algorithm, is indicated by FDR<0.01 (\*\*\*) and FDR<0.05 (\*\*). 

```{r scrnaseqnk, eval = T, out.width='0.9\\linewidth', echo=FALSE, fig.cap="(ref:scrnaseqnk-cap)", fig.scap = "scRNA-seq NK DEGs", out.extra=''}
knitr::include_graphics("./Figures/scRNAseq/nk_ipfhp_v1.pdf")
```

We also identified a number of differentially expressed genes in monocytes. \textit{TMEM176B} and its paralog \textit{TMEM176A} were upregulated in IPF samples across classical, intermediate, and non-classical monocyte subsets (Figure \@ref(fig:scrnaseqmono)). A similar upregulation in expression of \textit{HLA-DRB5} and \textit{HLA-DQA2} was observed in IPF monocytes, while classical and intermediate monocytes in HP were upregulated in \textit{CLEC4E} and \textit{FOLR3}. 

(ref:scrnaseqmono-cap) Differentially expressed genes between monocytes subsets as shown by violin plots (A) and UMAP plots (B). Significance, as determined by the MAST algorithm, is indicated by FDR<0.01 (\*\*\*).

```{r scrnaseqmono, eval = T, out.width='0.9\\linewidth', echo=FALSE, fig.cap="(ref:scrnaseqmono-cap)", fig.scap = "scRNA-seq NK DEGs", out.extra=''}
knitr::include_graphics("./Figures/scRNAseq/monocyte_ipfhp_v1.pdf")
```

## Discussion

In this chapter, we report blood single-cell transcriptional profiles of 11 patients with IPF or HP. Although the sample size is limited, single cell resolution highlights differences in both cellular abundance and gene expression between peripheral cell populations of ILD subtypes. To our knowledge, this is the first direct comparison between IPF and HP using scRNA-seq of PBMCs.

Unterman \textit{et al.} profiled cryopreserved PBMCs taken from 25 IPF patients (12 progressive, 13 stable) and 13 healthy controls using scRNA-seq and identified differences in cell abundance in classical monocytes (increased in IPF compared to controls) and total lymphocytes (decreased in IPF compared to controls, and decreased in progressive vs stable IPF), which provides context to our results: while classical monocytes are decreased in IPF compared to HP patients, these levels may still represent an increase in abundance compared to normal, and likewise with our observed increased in T lymphocyte subsets [@unterman_single-cell_2024]. Importantly, while fibrocytes are an intriguing candidate as progenitor cells for lung myofibroblasts due to their differentiation potential and higher circulation in IPF patients [@heukels_fibrocytes_2018], we were unable to identify a specific cluster of CD45\textsuperscript{+}COL1A1\textsuperscript{+} cells similar to the results reported by Unterman \textit{et al}. 

Within our cohorts, our analysis identified two clusters of classical monocytes, one of which had increased S100A8 and S100A12 expression and the other with increased HLA type II transcripts, which may represent a CD14\textsuperscript{+}HLA-DR\textsuperscript{hi} subset of monocytes [@evren_distinct_2021]. The C6 S100A8\textsuperscript{hi}S100A12\textsuperscript{hi}HLA\textsuperscript{lo} classical monocyte cluster was also identified by Unterman \textit{et al.} as being upregulated in IPF as well as COVID-19 patients [@unterman_single-cell_2024; @unterman_single-cell_2022], suggesting that this may be a specialized monocyte cohort in lung disease. In particular, S100A12 may be a biomarker of disease severity and prognosis in IPF [@li_s100a12_2022], and we have identified S100A12 as being upregulated in IPF whole blood compared to controls in our meta-analysis as well (Table \@ref(tab:ipfgenesblood)). Cluster C12 may represent a more differentiated monocyte given its low expression of S100A12 and increased expression of AHNAK, which is found in differentiated monocytes alongside increases in dysferlin expression resulting in increased adhesion [@de_morree_dysferlin_2013]. Moreover, the expression of F13A1 in monocytes has observed in infiltrating inflammatory monocytes in lung squamous cancer [@porrello_factor_2018], which promote fibrin cross-linking within the lung. 

We identified a number of differentially expressed genes between cell clusters in this cohort. We confirmed that none of our reported genes were correlated with the percentage of mitochondrial gene expression within each cell, which would otherwise suggest the results were confounded by cells undergoing apoptosis (Figure \@ref(fig:mtcorr)). The most striking difference was the overwhelming increase in TMEM176B and TMEM176A expression in all monocyte subsets of IPF compared to HP. TMEM176B was first discovered in fibroblasts in fibrotic lung tissue sections [@lurton_isolation_1999], and a follow-up paper identified their expression by fibroblasts in liver and kidney as well [@etikala_lr8_2017]. It is a marker of cell maturity as its expression decreases following activation in macrophges, dendritic cells, and monocyte-derived dendritic cells [@louvet_identification_2005]. A well-studied function of TMEM176B is its inhibition of the NLRP3 inflammasome via cytosolic Ca\textsuperscript{2+} regulation [@segovia_targeting_2019]. The NLRP3 inflammasome is a multi-protein complex that activates the innate immune system through production of IL-1\beta and IL-18, and macrophages obtained from bronchoalveolar lavage fluid (BALF) of IPF patients are shown to have impaired NLRP3 activation compared to RA-ILD and controls [@lasithiotaki_nlrp3_2016]. MYOM2 (myomesin-2) was also increased in NK and NKT cells, and this is a marker of NK cell maturity and cytotoxicity [@zhang_ezh2_2021; @cai_single-cell_2020]. Interestingly, we observed dichotomous expression of the HLA-E receptor KLRC1 (NKG2A) and KLRC2 (NKG2C) in HP and IPF, respectively. Binding of NKG2A results in an inhibitory effect on the cytotoxic activity of NK cells, thereby preventing cell lysis, but NKG2C binding results in activation of NK cells [@braud_hla-e_1998]. Taken together, this suggests that innate immunity may be inhibited in IPF, and NK cells have increased cytotoxic potential. 

In addition to a small sample size, our cohort is limited by its lack of progression data. Because of the prospective nature of this cohort, data regarding changes in FVC and D\textsubscript{LCO} over time are incomplete, and certain patients have been lost to follow-up. Nevertheless, some of our findings have been reported in IPF PBMCs, with FOLR3 in monocytes and MYOM2 in NK cells associated with progression in IPF, while TMEM176B, TMEM176A, and HLA-DQA2 in monocytes associated with stable IPF [@unterman_single-cell_2024]; these progressive genes match our findings in HP cell clusters, while the stable genes match findings in our IPF cohort. 

Future work on this project will profile more ILD subtypes and incorporate data relating to progression and radiological patterns, the latter of which was not possible in this cohort as all IPF patients had a UIP pattern, and all HP patients had a fHP pattern. In the realm of ILD diagnostics, we report numerous cell subsets with differentially expressed genes that have previously been investigated in fibrotic disease, particularly TMEM176B. Interestingly, when we examined differential expression of scRNA-seq DEGs in the [bulk blood RNA-seq](#bloodrnaseq) data, none were significant between IPF and HP, though TMEM176B was higher in IPF patients at a nominal \textit{p}-value of 0.05 (Figure \@ref(fig:bulkscrnaseq)). From our [pilot study of blood biomarkers](#bloodpilot), LTK was identified as a cluster marker of MAIT cells and VCAN for classical monocytes (Table \@ref(tab:clustermarkertab)), thus illustrating the benefits of scRNA-seq in pinpointing potential cellular effectors of disease. 
